dash/webapp/pages/prismic-linting.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect, FunctionComponent } from 'react';
+import Head from 'next/head';
+import styled from 'styled-components';
+import Header from '../components/Header';
+
+// Renders a date in the local timezone, including day of the week.
+// e.g. "Fri, 22 May 2020"
+const dateFormatter = new Intl.DateTimeFormat([], {
+  year: 'numeric',
+  month: 'long',
+  day: 'numeric',
+  weekday: 'short',
+});
+
+// Renders an HH:MM time in the local timezone, including timezone info.
+// e.g. "12:17 BST"
+const timeFormatter = new Intl.DateTimeFormat([], {
+  hour: 'numeric',
+  minute: 'numeric',
+  timeZoneName: 'short',
+});
+
+// Given an ISO 8601 date string, render it as a more friendly date
+// in the user's timezone.
+//
+// Examples:
+// - "today @ 12:00 BST"
+// - "yesterday @ 11:00 CST"
+// - "Fri, 22 May 2020 @ 10:00 PST"
+//
+// From https://alexwlchan.net/2020/05/human-friendly-dates-in-javascript/
+function getHumanFriendlyDateString(iso8601_date_string: string): string {
+  const date = new Date(Date.parse(iso8601_date_string));
+
+  // When are today and yesterday?
+  const today = new Date();
+  const yesterday = new Date().setDate(today.getDate() - 1);
+
+  // We have to compare the *formatted* dates rather than the actual dates --
+  // for example, if the UTC date and the localised date fall on either side
+  // of midnight.
+  if (dateFormatter.format(date) === dateFormatter.format(today)) {
+    return 'today @ ' + timeFormatter.format(date);
+  } else if (dateFormatter.format(date) === dateFormatter.format(yesterday)) {
+    return 'yesterday @ ' + timeFormatter.format(date);
+  } else {
+    return dateFormatter.format(date) + ' @ ' + timeFormatter.format(date);
+  }
+}
+
+const fontFamily = 'Gadget, sans-serif';
+const Issue = styled.div<{ type: string }>`
+  padding: 12px;
+  margin: 12px 0px;
+  ${props =>
+    props.type === 'error'
+      ? `
+      border: 1px solid rgba(224, 27, 47, 1);
+      background: rgba(224, 27, 47, 0.25);
+    `
+      : ''}
+
+  // This is the validation green from the global palette
+  ${props =>
+    props.type === 'success'
+      ? `
+          border: 1px solid rgba(11, 112, 81, 1);
+          background: rgba(11, 112, 81, 0.1);
+        `
+      : ''}
+`;
+
+const Index: FunctionComponent = () => {
+  const [resultsList, setResultsList] = useState(null);
+
+  useEffect(() => {
+    fetch('https://dash.wellcomecollection.org/prismic-linting/report.json?t=1')
+      .then(resp => resp.json())
+      .then(json => setResultsList(json));
+  }, []);
+
+  return (
+    <>
+      <Head>
+        <title>Prismic linting dashboard</title>
+      </Head>
+      {resultsList && (
+        <div
+          style={{
+            fontFamily,
+          }}
+        >
+          <Header title="Prismic linting" />
+          <div
+            style={{
```

Comment:
This being dash I don't mind, really, but we could align to a styled component too.

Reply:
Gonna merge as-is for now, but yeah as we lean more on dash we should be stricter about code quality.